new-site/gatsby-config.js

```diff
@@ -198,5 +198,15 @@ module.exports = {
     // this (optional) plugin enables Progressive Web App + Offline functionality
     // To learn more, visit: https://gatsby.dev/offline
     // `gatsby-plugin-offline`,
+    {
+      resolve: 'gatsby-plugin-matomo',
+      options: {
+        siteId: '831',
+        matomoUrl: 'https://webanalytics.digiaiiris.com/js',
+        siteUrl: 'https://hds.hel.fi',
+        matomoJsScript: 'piwik.min.js',
+        matomoPhpScript: 'tracker.php',
+      }
```

Comment:
Just out of curiosity, where do these values come from?

Reply:
From matomo site: https://digiaiiris.com/shibboleth-ds/index.html?entityID=https%3A%2F%2Fdigiaiiris.com%2Fshibboleth&return=https%3A%2F%2Fdigiaiiris.com%2FShibboleth.sso%2FLogin%3FSAMLDS%3D1%26target%3Dss%253Amem%253Aae3a04ba89db20199a399ca4ed855884ac5c5f7dbce55491cc96a4b111a9fcc1

You need to select Helsinki Design System and then click "Analytiikka matomossa".